pkg/operator/ceph/cluster/osd/osd_pvc_test.go

```diff
@@ -282,13 +282,30 @@ func testOSDsOnPVC(t *testing.T) {
 	}
 
 	var c *Cluster
+	var provisionDone bool = false
+	waitForDone := func() {
+		for {
+			if provisionDone {
+				infof(t, "provisioning done")
```

Comment:
(unrelated directly to this PR) Do we really need the `infof()` helper? It just prints the test name as a prefix? If someone wants to know the test that is running they can run `go test -v`. Would that work instead?

Reply:
For me, it's more about differentiating log output that comes from the test framework and log output that comes from the Rook code. Being able to use `t.Log` and use `-v` to interleave the output might be just as good though. I'll investigate doing that.